src/run/parser/time_split_tracker.rs

```diff
@@ -124,7 +128,8 @@ pub fn parse<R: BufRead>(source: R, path_for_loading_other_files: Option<PathBuf
     parse_history(&mut run, path).ok();
 
     for comparison in comparisons {
-        run.add_custom_comparison(comparison);
+        let _ = run.add_custom_comparison(comparison)
+            .map_err(|_| Error::InvalidComparisonName);
```

Comment:
The error is ignored here as well, but actually in cases like this (i.e third party format, which doesn't have these rules) it even should be ignored (or maybe even better: renamed). So then you don't need the InvalidComparisonName error.

Reply:
Now when you say "renamed" do you mean the new comparison name, so that it adds it regardless?

Reply:
Yeah, so that we can parse foreign formats that don't have these rules regardless.

Reply:
However you may need a proper Error type on add_custom_comparison then, that lets you distinguish a double add vs. a [Race] name